gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -50,6 +50,7 @@ ext.externalDependency = [
     "guava": "com.google.guava:guava:15.0",
     "groovy": "org.codehaus.groovy:groovy:2.4.8",
     "gson": "com.google.code.gson:gson:2.6.2",
+    "gsonJavatimeSerialisers": "com.fatboyindustrial.gson-javatime-serialisers:gson-javatime-serialisers:1.1.1",
```

Comment:
does this dependency require us to add a repository?

Reply:
nope, it's in the standard maven repo - https://search.maven.org/artifact/com.fatboyindustrial.gson-javatime-serialisers/gson-javatime-serialisers/1.1.1/jar